Android build issue

Trying to build webRelease with gradle. I was able to generate an apk before, but now I get this error with gradle. 

/Users/USER/Desktop/map_app/omim/map/cloud.cpp:36:32: error: use of undeclared identifier 'CLOUD_URL'
std::string const kServerUrl = CLOUD_URL;

Any ideas?

--- a/configure.sh
+++ b/configure.sh
@@ -92,6 +92,7 @@ else
 #define PASSPORT_URL ""
 #define PASSPORT_APP_NAME ""
 #define UGC_URL ""
+#define CLOUD_URL ""
 #define MAX_PARTNERS_COUNT 0
 #define PARTNER1_NAME ""
 #define PARTNER1_BANNER_PLACEMENT_ID ""

